lib/knackhq/client.rb

```diff
@@ -48,10 +48,9 @@ def records_by_page(object, page_number)
                      .records
                      .get(:params => { :page => page_number,
                                        :rows_per_page => 500 })
-                     .to_h
-      return [] if hash_request.empty?
-      transform_hash_keys = hash_request[:records]
-      symbolize_hash_keys!(transform_hash_keys)
+      payload = payload_hash(hash_request)
+      return [] if payload.empty?
+      payload[:records]
     end
 
     def records_info(object)
```

Comment:
I suppose I could also remove the three records methods and combine them into one master record method:
```Ruby
    def records(object, options ={})
      options[:rows_per_page] ||= 500
      options[:page_number] || 1
      hash_request = ...
    end
# which will return
{:total_pages=>999, :current_page=>1, :total_records=>1000000 :records=>[record1, record2, etc..]}
```